src/pages/index.js

```diff
@@ -143,5 +224,16 @@ const addCardFormValidator = new FormValidator(
   constants.settings,
   addNewCardForm
 );
+const updateAvatarFormValidator = new FormValidator(
+  constants.settings,
+  updateAvatarImg
+);
+
+const updateProfileNameValidator = new FormValidator(
+  constants.settings,
+  updatePrfileNameDesc
```

Comment:
`updateProfileNameValidator` is not needed because you have `editProfileFormValidator` as the profile form validator